go.mod

```diff
@@ -4,102 +4,110 @@ replace github.com/grpc-ecosystem/grpc-gateway => github.com/TheThingsIndustries
 
 replace github.com/robertkrimen/otto => github.com/TheThingsIndustries/otto v0.0.0-20181129100957-6ddbbb60554a
 
-replace github.com/labstack/echo/v4 => github.com/TheThingsIndustries/echo/v4 v4.0.1-0.20190409124425-ee570f243713
+// Fix versions of golang.org/x modules
```

Comment:
It's quite clear that you `Fix versions of golang.org/x modules` here, can you add reasoning *why* you we need these replacements?